__tests__/api/request/support.test.ts

```diff
@@ -172,6 +178,9 @@ function runEmailAPITests() {
         supportType: "support",
         name: "name",
         email: "email@email.com",
+        department: "Department",
+        branch: "Branch",
+        jobtitle: "Job Title",
```

Comment:
A bit of a nit: the same two word variable above (`supportType`) uses camel case format and `jobtitle` is all lowercase

Reply:
Sure, sounds good